### ISSUE TITLE
error with specific environments after update to 0.15.13 

### ISSUE THREAD

**Issue by LKuechler** [sinnerschrader/patternplate#22]
The build throws the error that in the generated "@environments/special/index.less" the variable "@ns" would be not defined. The file seems to miss the defined globalVars.
This is not happening with environments that apply on all patterns. 

```javascript
{
  "name": "special",
  "version": "1.0.0",
  "include": ["special/cultures/**"],
  "apply": ["special"],
  "environment": {
    "transforms": {
      "less": {
        "opts": {
          "globalVars": {
            "ns": "prefix-"
          }
        }
      },
      "react": {
        "opts": {
          "globals": {
            "context": {
              "ns": "prefix-"
            }
          }
        }
      }
    }
  }
}
```

**Comment by marionebl** [comment]
Hey @LKuechler, I investigated this with the code samples provided off the record and found this is caused by misconfiguration - for including and excluding you'll have to use fully qualified glob patterns, e.g. `['**/*', '!special/**/*']`. Anyway, the system could be more helpful with giving hints about bad `applyTo` config, I'll open a seperate issue for this

**Comment by marionebl** [comment]
Mea culpa, this is indeed a problem with the `build-bundles` task. I'll publish a fix for this any minute now.

**Comment by marionebl** [comment]
@LKuechler, released with 0.14.9 - could you retest and close if fixed?

**Comment by marionebl** [comment]
Solved
